Makefile

```diff
@@ -192,33 +194,33 @@ ifeq ($(BUILD_ENTERPRISE_READY),true)
 endif
 
 i18n-extract: ## Extract strings for translation from the source code
-	env GO111MODULE=off go get -u github.com/mattermost/mattermost-utilities/mmgotool
-	$(GOPATH)/bin/mmgotool i18n extract
+	cd tools && $(GO) install -mod=readonly github.com/mattermost/mattermost-utilities/mmgotool && cd ..
```

Comment:
Why do we need to run `install`? Is `build` not enough? This way we can place the tools where ever we want.

Reply:
Yeah we can, but `go install` feels like a more idiomatic way to do this. With `go build`, we would need to specify `-o` every time. But with this, we are controlling it just by setting the `GOBIN` which seems like a more appropriate way to achieve this. 3/5.